deployments/values/supported-matrix.yaml

```diff
@@ -239,13 +239,13 @@ matrix:
       version: "2.0.3"
       pattern: *ga_hotfixes_pattern
     search-enterprise:
-      version: "3.1.1"
+      version: "3.1"
       pattern: *ga_hotfixes_pattern
```

Comment:
please explain

Reply:
https://docs.alfresco.com/content-services/7.2/support/ It says there thats its 3.1